### PR TITLE
Added Github Actions config file

### PR DESCRIPTION
**Pull request by imnotteixeira**
Introducing Github Actions to the workflow!

Sorry, Travis, but we have to let you go.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Launch the stack and run the tests
+      run: docker-compose up --exit-code-from test test
+
+    - name: Audit
+      run: npm run audit
```
Comment (andrefmrocha) [inline .github/workflows/ci.yml:26]: Missing newline at end of file

**Comment by andrefmrocha** [comment]
What are the particular reasons for you to be leaving travis? I know that its time for builds for open source have been significantly shortened but I was looking to see if there was any other reason

**Comment by imnotteixeira** [comment]
> What are the particular reasons for you to be leaving travis? I know that its time for builds for open source have been significantly shortened but I was looking to see if there was any other reason

It has been failing too much lately when fetching images from docker repos, also staying in github is a lot nicer when reviewing stuff :P

**Comment by imnotteixeira** [comment]
> > What are the particular reasons for you to be leaving travis? I know that its time for builds for open source have been significantly shortened but I was looking to see if there was any other reason
> 
> It has been failing too much lately when fetching images from docker repos, also staying in github is a lot nicer when reviewing stuff :P

image
 
Just in time.... :P

**Comment by andrefmrocha** [comment]
> > > What are the particular reasons for you to be leaving travis? I know that its time for builds for open source have been significantly shortened but I was looking to see if there was any other reason
> > 
> > 
> > It has been failing too much lately when fetching images from docker repos, also staying in github is a lot nicer when reviewing stuff :P
> 
> image
> 
> Just in time.... :P

But this is related to the pull rate limit on their nodes, is this the error you were mentioning?

**Comment by imnotteixeira** [comment]
> > > > What are the particular reasons for you to be leaving travis? I know that its time for builds for open source have been significantly shortened but I was looking to see if there was any other reason
> > > 
> > > 
> > > It has been failing too much lately when fetching images from docker repos, also staying in github is a lot nicer when reviewing stuff :P
> > 
> > 
> > image
> > Just in time.... :P
> 
> But this is related to the pull rate limit on their nodes, is this the error you were mentioning?

Yes, recently they increased limitations afaik, however in Circle I haven't seen it happen

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-
-sudo: required
-
-services:
-  - docker
-
-env:
-  global:
-    # The hostname is set to mongo because of the docker configuration (using networks to access the DB)
-    - DB_HOSTNAME=mongo
```
Comment (andrefmrocha) [inline .travis.yml:14]: Is this env var no longer needed?

**Review by andrefmrocha — APPROVED** [review]
Lgtm! Just have a small question

File: .travis.yml (hunk 2 above)
Comment (imnotteixeira) [reply .travis.yml:14]: Since the tests are passing and I can't find any reference to that in the code, I assume it's no longer needed

**Pull request merged**